app/views/cohorts/client_notes/index.haml

```diff
@@ -1,3 +1,5 @@
+= render 'cohorts/breadcrumbs'
```

Comment:
Correct. I included this when I was making sure I knew which controller went where, but I left it in case we decide to go back to sending clients here instead of client/notes, or for the case where the modal is opened in a separate view (I sometimes do this in development, so I can imagine somebody sending an link to somebody else).